documentation/docs/validator-zone/external-chains/polygon.md

```diff
@@ -0,0 +1,184 @@
+# Set up your Polygon Mumbai Testnet node
+-----------
+## Disclaimer
+!> :fire: Please note that the guide below may not always be up to date with the latest version or updates from the project, so make sure you always check the official documentation first and run the latest version of software and required dependencies. If you find a problem, please submit an issue to this repository following the template.
+
+
+## Prerequisites
+- Complete all steps from [Setup with Docker](/setup/setup-with-docker.md) or [Setup with Binaries](/setup/setup-with-binaries.md)
+- Minimum hardware requirements: 4-8+ core CPU , 16-32GB RAM, 100GB+ free storage space.
```

Comment:
Correct me if I am wrong but polygon would take much more than 100GB+ right ?

Reply:
> Correct me if I am wrong but polygon would take much more than 100GB+ right ?

22G     ./.heimdalld/data
54G     ./.bor/data/bor/chaindata

I've used snapshots and I have it in notes as well but not sure why it wasn't included in the merge. Will add the note, thanks.

Reply:
Got it - I guess I had downloaded the archive one which was much more than this iirc

Reply:
Added Polygon fix:
https://github.com/axelarnetwork/axelarate-community/pull/282/commits/1f02df7b593b744b02c044aa17e8c14b888df541